.gitignore

```diff
@@ -14,6 +14,7 @@
 .sw[nop]
 *.tmp
 *.coverage.*
+pytest-report.html
```

Comment:
yes, but it's useful to see locally and if it's a hidden file it'll be harder for people to find. I could go either way on this - just noting.